install.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+if [[ "$OSTYPE" == "linux-gnu" ]]; then
+        chmod +x ./scripts/ubuntu.sh
+        ./scripts/ubuntu.sh
+elif [[ "$OSTYPE" == "darwin"* ]]; then
+        chmod +x ./scripts/macosx.sh
+        ./scripts/macosx.sh
+else
+        echo "Unhandled OS type"
```

Comment:
This looks great! Let's update this message to something more conversational, like "Sorry, this installation script only works on Mac OS X and Ubuntu Linux. Looks like your operating system is $OSTYPE."

Reply:
That is friendlier :)